redis/cluster.py

```diff
@@ -1154,19 +1161,13 @@ def _execute_command(self, target_node, *args, **kwargs):
                     connection.send_command("ASKING")
                     redis_node.parse_response(connection, "ASKING", **kwargs)
                     asking = False
-                response_from_cache = connection._get_from_local_cache(args)
```

Comment:
`RedisCluster` client cannot handle MOVED responses from server and consider this as an error. We're unable to reproduce this test case as the exception will be thrown as soon as failover will be triggered. Here's a statement from documentation:

`However, if the command execution fails due to cluster topology changes, a retry attempt will not be made, since the passed target node/s may no longer be valid, and the relevant cluster or connection error will be returned.`

https://redis-py.readthedocs.io/en/stable/clustering.html

Reply:
@dmaier-redislabs UPD: Even with the limitation that we already have I was able to test client as if it's handle MOVED errors. The outcome was exactly as you mentioned, we will run into data duplication on caches for node1 and node2.

For now updated RedisCluster client to have a single cache per all nodes